ui/README.md

```diff
@@ -29,15 +29,6 @@ cd ui/
 pnpm test
 ```
 
-### VSCode
-
-The Jest extension is your friend. You will want to configure the following in your `settings.json`:
-
-```
-"jest.rootPath": "ui",
-"jest.jestCommandLine": "node_modules/.bin/jest --config jest.config.js",
```

Comment:
Couldn't quite get this extension wring with our new setup so just removing this for now